docs/getting_started/online_comp_getting_started.md

```diff
@@ -0,0 +1,84 @@
+
+Similar to the [navigation tree](/handbook/mbed-Compiler-Getting-Started#navigation-tree), the browse list allows HTML5 drag and drop upload of files and zip archives. Because you can't upload just anywhere in your workspace and the uploaded files exist under the browse list parent, we recommend you use the navigation tree instead.
+
+The **sidebar panel** has two tabs - Summary and Build. The **Summary** tab offers extended information about program or library creation date, last modified date, tags and description. It also includes [collaboration](/handbook/Collaboration)-related information, such as public repository URL and ability to update and publish to and from the public repository. The **Build** tab is essential for successfully built programs. It offers valuable statistics about the Flash and RAM usage.
+
+###Editor
+The editor is the main tool of the mbed Online Compiler. It's a fully featured, lightweight code editor that uses high standard syntax highlighting, keyboard [shortcuts](/online_comp.mds#keyboard-shortcuts) and undo and redo operations. It's also compliant with all major browsers' cut, copy and paste text operations, so you can inject and extract code between your host operating system and the mbed editor.
+
+![](/images/editor-overview.png) 
+
+The editor features include:
+ * Standard C/C++ syntax highlighting.
+ * Python, Lua, JavaScript, XML, HTML, CSS syntax highlighting.
+ * Unlimited undo/redo operations buffer.
+ * Compliant clipboard text operations - cut, copy and paste.
+ * "Artistic Style" code formatting for text selections or whole file.
+ * Find text with case, word and regular expression matching.
+ * Find in Files with filtering capabilities.
+ * Mouse, keyboard, [touch or tap](/handbook/Guide-to-mbed-Compiler-on-tablet-device) navigation.
+ * Text selection block comment and code indention manipulation.
+ * UTF-8/Unicode base encoding.
+ * Print-friendly code preview.
+
+Additionally, you can explore the editor [shortcuts and controls](/online_comp.md#keyboard-shortcuts) section, where you can find more information on shortcuts and hidden functions.
+
+###Output panel
+The output panel is under the editor (or browser panel), and its main purpose is to display the output of the ARMCC compile engine when you decide to compile code. 
+
+A typical compile error looks like the image below:
+
+![](/images/compile-output.png)
+
+If you click the number under the "Error Number" column, a wiki page opens with more information about the compile error or warning. You can also double-click the text under "Location" and open the source file at the line or column of the definition.
+
+The output panel also displays results for the "Find in Folder" functionality. Having the find result under the editor lets you quickly switch between matches in different files.
```

Comment:
Query: Is "Find in Folder" correct, or is it "In Folder" or "Find Results" or something else?